thirdparty/openapi3/components/schemas/AuthenticationValue.yaml

```diff
@@ -3,7 +3,6 @@ anyOf:
   - $ref: '../../../../fspiop/v1_0/openapi3/components/schemas/OtpValue.yaml'
   - $ref: '../../../../fspiop/v1_1/openapi3/components/schemas/QRCODE.yaml'
   - $ref: ./U2FPinValue.yaml
```

Comment:
if we are removing the pattern then the size of the property is unlimited so it could bring some security issues, let put min & max boundaries for string.

Reply:
OtpValue and QRCODE contain their own character limiting patterns or min/max.
Since U2FPinValue is an object I'm pretty sure putting mix/max will cause `ajv` to fail validation.